# <center> Поведение городского транспорта в городе Сан-Паулу </center>

Сслыка на данные и дополнительную информкацию: https://archive.ics.uci.edu/ml/machine-learning-databases/00483/

Информация о наборе данных:
База данных была создана с записями поведения движения транспорта города Сан-Паулу в Бразилии с 14 декабря 2009 года по 18 декабря 2009 года (с понедельника по пятницу). Регистрируется с 7:00 до 20:00 каждые 30 минут. Набор данных «Поведение городского движения в городе Сан-Паулу в Бразилии» использовался в научных исследованиях в Университете Нове-де-Жюльо - аспирантура по информатике и управлению знаниями.

In [1]:
import pandas as pd 
dataset =pd.read_csv("Behavior of the urban traffic of the city of Sao Paulo in Brazil.csv",sep=";")

In [4]:
dataset.head().T

0    1    2    3     4
Hour (Coded)                             1    2    3    4     5
Immobilized bus                          0    0    0    0     0
Broken Truck                             0    0    0    0     0
Vehicle excess                           0    0    0    0     0
Accident victim                          0    0    0    0     0
Running over                             0    0    0    0     0
Fire vehicles                            0    0    0    0     0
Occurrence involving freight             0    0    0    0     0
Incident involving dangerous freight     0    0    0    0     0
Lack of electricity                      0    0    0    0     0
Fire                                     0    0    0    0     0
Point of flooding                        0    0    0    0     0
Manifestations                           0    0    0    0     0
Defect in the network of trolleybuses    0    0    0    0     0
Tree on the road                         0    0    0    0     0
Semaphore off                            0    0    0    0     0
Intermittent Semaphore                   0    0    0    0     0
Slowness in traffic (%)                4,1  6,6  8,7  9,2  11,1

In [5]:
dataset.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 135 entries, 0 to 134
Data columns (total 18 columns):
Hour (Coded)                             135 non-null int64
Immobilized bus                          135 non-null int64
Broken Truck                             135 non-null int64
Vehicle excess                           135 non-null int64
Accident victim                          135 non-null int64
Running over                             135 non-null int64
Fire vehicles                            135 non-null int64
Occurrence involving freight             135 non-null int64
Incident involving dangerous freight     135 non-null int64
Lack of electricity                      135 non-null int64
Fire                                     135 non-null int64
Point of flooding                        135 non-null int64
Manifestations                           135 non-null int64
Defect in the network of trolleybuses    135 non-null int64
Tree on the road                         135 non-null i

#### Количество столбцов: 18.
#### Количество строк: 135.
#### Пропусков нет.
#### Информация о признаках:
1.	Hour - Время {1 - 7:00, 2 - 7:30, 3 - 8:00, 4 - 8:30, 5 - 9:00, 6 - 9:30, 7 - 10:00, 8 - 10:30, 9 - 11:00, 10 - 11:30, 11 - 12:00, 12 - 12:30, 13 - 13:00, 14 - 13:30, 15 - 14:00, 16 - 14:30, 17 - 15:00, 18 - 15:30, 19 - 16:00, 20 - 16:30, 21 - 17:00, 22 - 17:30, 23 - 18:00, 24 - 18:30, 25 - 19:00, 26 - 19:30, 27 - 20:00}, тип int64.
2.	Immobilized bus - иммобилизированные автобусы, тип int64.
3.	Broken Truck - сломанные грузовики, тип int64.
4.	Vehicle excess - избыток автомобилей, тип int64.
5.	Accident victim - жертвы аварий, тип int64. 
6.	Running over - бег, тип int64.
7.	Fire Vehicles - пожарная техника, тип int64.
8.	Occurrence involving freight - участие груза, тип int64.
9.	Incident involving dangerous freight - инциденты с опасным грузом, тип int64.
10.	Lack of electricity - недостаток электричества, тип int64.
11.	Fire - пожар, тип int64.
12.	Point of flooding - точка наводнения, тип int64.
13.	Manifestations - проявления, тип int64.
14.	Defect in the network of trolleybuses - дефекты в сети троллейбусов, тип int64.
15.	Tree on the road - деревья на дороге, тип int64.
16.	Semaphore off - выключение семафора, тип int64.
17.	Intermittent Semaphore - прерывание семафора, тип int64.
18.	Slowness in traffic (%) (Target) - процентное значение замедления движения (целевая переменная), тип object.
#### Характеристики данных: Многомерные, временные ряды.
#### Применимы для задач классификации, регрессии.